src/engine/experimental/capabilities.md

```diff
@@ -0,0 +1,37 @@
+# Engine API -- Capabilities
+
+Specification of `engine_exchangeCapabilities` method exchanging with a list of Engine API methods supported by the server (execution layer client) and the client (consensus layer client) down to a version of each method.
+
+The proposed method should become a part of [`common.md`](../common.md) document if accepted.
+
+## Methods
+
+### engine_exchangeCapabilities
+
+*Note:* The method itself doesn't have a version suffix.
+
+#### Request
+
+* method: `engine_exchangeCapabilities`
+* params:
+    1. `Array of string` -- Array of strings, each string is a name of a method supported by consensus layer client software.
+* timeout: 1s
+
+#### Response
+
+`Array of string` -- Array of strings, each string is a name of a method supported by execution layer client software.
+
+#### Specification
+
+1. Consensus and execution layer client software **MUST** exchange with a list of currently supported Engine API methods down to a version of each method. Consider the following examples:
```

Comment:
The call to getCapabilities is strictly redundant / there's no real need to call it for communication to work and **MUST** makes it impossible to restart an execution client without notifying the CL - ie the CL cannot generally detect that the EL went down, meaning both EL and CL must support this function not having been called at all.

Reply:
I've made it in a bit different way. But the first statement in the spec is clearly saying that this method may be used and that EL must not surface any error messages based on the time when this method was or was not called. Hopefully, this addresses the concern.